web/packages/teleport/src/Console/Console.tsx

```diff
@@ -78,22 +90,32 @@ export default function Console() {
 
   return (
     <StyledConsole>
-      <Flex bg={colors.terminalDark} height="32px">
-        <Tabs
-          flex="1"
-          items={documents}
-          onClose={onTabClose}
-          onSelect={onTabClick}
-          activeTab={activeDocId}
-          clusterId={clusterId}
-          disableNew={disableNewTab}
-          onNew={onTabNew}
-        />
-        <ActionBar onLogout={onLogout} />
-      </Flex>
-      {$docs}
-      {hasSshSessions && (
-        <AjaxPoller time={POLL_INTERVAL} onFetch={onRefresh} />
+      {attempt.status === 'failed' && <Danger>{attempt.statusText}</Danger>}
```

Comment:
if you look at how desktop session handles init error (can be found in storybook `DesktopSession`), it's a dialog telling the user to try refreshing, we should do same?

Reply:
I'm not sure a full dialog would be needed. I can see adding a static text in `Danger` of something like "User context could not be found. Refresh the page to try again". My reasoning is
1. technically, everything about Console will still work without this user context EXCEPT for the upcoming moderated file transfers, which is a low-use feature. So a small error should be fine
2. I don't _think_ we'd ever get to the `Console` without already having a valid user so this could generally be fixed with a quick refresh anyway. 

so maybe just this
```js
<Danger>Could not fetch user information. Please refresh to try again.</Danger>
```

Thoughts?